helmfile.d/charts/grafana-dashboards/dashboards/capacity-management-dashboard.json

```diff
@@ -18,1134 +18,1379 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 1,
+  "id": 7,
   "links": [],
-  "liveNow": false,
   "panels": [
     {
+      "collapsed": true,
```

Comment:
Collapsing a row moves panels around in the JSON, making it harder to see what's changed. I'd suggest reverting that.